packages/apollos-data-prayer/src/prayer-menu-categories/data-source.js

```diff
@@ -10,6 +10,7 @@ class PrayerMenuCategory extends ContentItem.dataSource {
       .filter(
         `ContentChannelId eq ${ROCK_MAPPINGS.PRAYER_MENU_CATEGORIES_CHANNEL_ID}`
       )
+      .orderBy(`Priority`)
```

Comment:
let's try the drag and drop instead of adding this to the code

Reply:
👍